spark/src/main/scala/geotrellis/spark/ingest/Ingest.scala

```diff
@@ -61,15 +61,14 @@ object Ingest {
       cacheLevel: StorageLevel = StorageLevel.NONE,
       resampleMethod: ResampleMethod = NearestNeighbor
     )
-    (sink: (RasterRDD[K], Int) => Unit)
-    (implicit tiler: Tiler[T, K, Tile]): Unit =
+    (sink: (RasterRDD[K], Int) => Unit): Unit =
   {
     sourceTiles.persist()
     val reprojectedTiles =
       sourceTiles.reproject(destCRS, resampleMethod).cache()
     val (zoom, rasterMetaData) =
       RasterMetaData.fromRdd(reprojectedTiles, destCRS, layoutScheme)(_.projectedExtent.extent)
-    val tiledRdd = tiler(reprojectedTiles, rasterMetaData, resampleMethod).cache()
+    val tiledRdd = reprojectedTiles.cutTiles(rasterMetaData, resampleMethod).cache()
```

Comment:
Good catch, that should be a `tileToLayout` call

On Sat, Jan 16, 2016 at 5:47 PM, Eugene Cheipesh <notifications@github.com>
wrote:

> In spark/src/main/scala/geotrellis/spark/ingest/Ingest.scala
> <https://github.com/geotrellis/geotrellis/pull/1285#discussion_r49938387>:
>
> >    {
> >      sourceTiles.persist()
> >      val reprojectedTiles =
> >        sourceTiles.reproject(destCRS, resampleMethod).cache()
> >      val (zoom, rasterMetaData) =
> >        RasterMetaData.fromRdd(reprojectedTiles, destCRS, layoutScheme)(_.projectedExtent.extent)
> > -    val tiledRdd = tiler(reprojectedTiles, rasterMetaData, resampleMethod).cache()
> > +    val tiledRdd = reprojectedTiles.cutTiles(rasterMetaData, resampleMethod).cache()
>
> Isn't this missing something that will .reduceByKey on the cut tiles?
> It looks like this could a call to tileToLayout.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geotrellis/geotrellis/pull/1285/files#r49938387>.
>